src/Compilers/VisualBasic/Test/Syntax/Parser/ParserRegressionTests.vb

```diff
@@ -918,20 +919,8 @@ End Enum
     End Sub
 
     <WorkItem(2867, "https://github.com/dotnet/roslyn/issues/2867")>
-    <Fact(Skip:="https://github.com/dotnet/roslyn/issues/7159")>
     Public Sub TestBinary()
-        ' use a fixed seed so the test Is reproducible
-        Dim random = New System.Random(12345)
-        Const n As Integer = 40 * 1000 * 1000 ' 40 million "character"s
-        Dim builder = New System.Text.StringBuilder(n + 10)
-        For i As Integer = 0 To n - 1
-            builder.Append(ChrW(random.Next(&HFFFF)))
-        Next
-
-        Dim source = builder.ToString()
-        Dim tree = VisualBasicSyntaxTree.ParseText(source)
-
-        Assert.Equal(source, tree.ToString())
+        Dim tree = VisualBasicSyntaxTree.ParseText(New RandomizedSourceText())
```

Comment:
The original issue was "we try to read binary files, and we generate millions of errors to report back to the user; this crashes us."  

Although this doesn't use the same seed, it will trigger the original issue.